proposals/bip-002.md

```diff
@@ -47,7 +47,9 @@ marking the incompleteness of this information to the UAST user.
 This change will add two new roles the UAST definition:
 
 - `Incomplete`: For nodes that have been annotated but the annotations don't
-  express the full semantic meaning.
+  express the full semantic meaning or there aren't any other annotations because
```

Comment:
A bit shorter and more clear maybe:
_For nodes that don't have their full semanting meaning annotated, including both partially annotated nodes and nodes without any other annotation_